lib/ansible/modules/windows/win_iis_virtualdirectory.ps1

```diff
@@ -65,7 +65,7 @@ try {
 
   # Remove directory
   If ($state -eq 'absent' -and $directory) {
-    Remove-Item $directory_path
+    Remove-Item $directory_path -Recurse
```

Comment:
Should also include `-Force` in case there are any hidden or read only files in the directory.